frontend/components/forms/_styles.scss

```diff
@@ -1,11 +1,13 @@
 .form {
+  // Used on login forms
   &__base-error {
     background-color: $ui-error;
     border-radius: 3px;
     color: $core-fleet-white;
     font-size: $x-small;
-    margin-top: 10px;
-    padding: 5px 0;
+    height: 16px;
+    margin: -12px 0;
```

Comment:
why the negative margin here?

Reply:
We use these old school inline errors instead of toast messages here, existing bug that makes the entire card/UI jump larger in height, this is to make it not as noticeable until a better design fix is prioritized.

See:

https://github.com/user-attachments/assets/6ac816fb-7f85-40d1-b127-3f9313219962